addon/components/ownership-selection-modal.js

```diff
@@ -25,6 +26,12 @@ export default Component.extend({
     groups: true
   },
 
+  _resetWindow: observer('entity.id', function() {
+    if (isEmpty(this.selectedUsers)) {
```

Comment:
this.set('currentWindow',  this.entity.ownedBy.startsWith('composite:') ? 'people' : 'groups')